pkg/engine2/testdata/k8s_api.dataflow-viz.yaml

```diff
@@ -4,11 +4,11 @@ resources:
     parent: eks_cluster/eks_cluster-0
 
 
-  load_balancer/rest-api-4-integ6897f0b9:
+  load_balancer/rest-api-4-integbcc77100:
```

Comment:
Is this non-deterministic?

Reply:
i dont think so, the ordering did change, which i assumed was from the way we load the graph, but i ran the same architecture 10 times and always saw the same result

Reply:
Just wondering if the UUIDs weren't deterministic, do we set the seed somewhere?

Reply:
dont think we do, we could probably look into that